frontend/.eslintrc.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
     'array-callback-return': 1,
     'consistent-return': 1,
     'default-case': 1,
-    'eqeqeq': 1,
+    eqeqeq: 1,
```

Comment:
I'm not sure why this eslint error is taking the quotes off `eqeqeq` and `camelcase` on line 46 :-/